docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: TraceQL
+menuTitle: TraceQL
+description: Learn about TraceQL, Tempo's query language for traces
+weight: 450
+aliases:
+  - /docs/tempo/latest/traceql/
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. Currently, TraceQL query can select traces based on the following see [future work](./architecture.md) for where the language is heading.
+
+- Span and resource attributes, timing, and duration
+- Basic aggregates: `count()` and `avg()`
+
+The TraceQL language uses similar syntax and semantics as [PromQL](https://grafana.com/blog/2020/02/04/introduction-to-promql-the-prometheus-query-language/) and [LogQL](https://grafana.com/docs/loki/latest/logql/), where possible. TraceQL recognizes two types of data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs.
+
+TraceQL requires Tempo’s Parquet columnar format to be enabled. For information on enabling Parquet, refer to the [Apache Parquet backend](https://grafana.com/docs/tempo/latest/configuration/parquet/) Tempo documentation.
+
+## TraceQL query editor
+
+With Tempo 2.0, you can use the TraceQL query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface. For more information, refer to [TraceQL query editor](query-editor.md).
+
+<p align="center"><img src="../assets/query-editor-http-method.png" alt="Query editor showing request for http.method" /></p>
+
+## Construct a TraceQL query
+
+In TraceQL, a query is an expression that is evaluated on one trace at a time. The query is structured as a set of chained expressions (a pipeline). Each expression in the pipeline selects or discards spansets from being included in the results set. For example:
+
+```
+{ span.http.status_code >= 200 && span.http.status_code < 300 } | count() > 2
+```
+
+<!--  jpe finish ... -->
```

Comment:
jpe comment. i think it's referring to the below paragraph needing attention.

Reply:
yes, the explanation of what the query on line 37 does.

Reply:
right, i just mean we should tighten up the description below and remove line 40.